WooCommerce/WooCommerceTests/Tools/ResultsControllerUIKitTests.swift

```diff
@@ -38,13 +44,17 @@ class ResultsControllerUIKitTests: XCTestCase {
             )
         }()
 
+        tableView = MockupTableView()
+        tableView.dataSource = self
+
         resultsController.startForwardingEvents(to: tableView)
-        try? resultsController.performFetch()
+        try! resultsController.performFetch()
```

Comment:
just curious, is this change intentional? I tried the old diff and the tests passed as well